### PR TITLE
[#686] Change names of buttons and headers

### PR DESCRIPTION
**Pull request by goreck888**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -96,6 +96,29 @@
     t.index ["service_id"], name: "index_offers_on_service_id"
   end
 
+  create_table "order_changes", force: :cascade do |t|
```
Comment (mkasztelnik) [inline db/schema.rb]: We don't have order_changes and orders models. Why this was added to this pr?

Comment (kmarszalek) [inline db/schema.rb]: what are these changes

**Comment by mkasztelnik** [comment]
Changelog entry is missing.

**Comment by roksanaer** [comment]
what about "Services" label under "My projects and orders". shouldn't it claim "orders" or "projects" to be consistent? @agpul did u check this?

**Comment by goreck888** [comment]
> Changelog entry is missing.

Now it's added

**Comment by goreck888** [comment]
> what about "Services" label under "My projects and orders". shouldn't it claim "orders" or "projects" to be consistent? @agpul did u check this?
> 
> edit: ok, i get it. this is a project name, named Services ;) this one will be removed probably in the foreseeable future. Are we ready to merge this?
> @goreck888 ? why don't u look after ur pr? :P ;)

I edited it exactly as it is in the ticket describes whole changes. The one thing I didn't make is one of the points because I didn't find any similar text like it is in the description.
Why do you think I don't look after, my branch is rebased, up to date and ready to merge :)

**Pull request merged**